### DISCUSSION TITLE
Is it possible to reuse annotations from an image to another image?

### DISCUSSION THREAD

**Discussion by vasigiurgi** [cvat-ai/cvat#2980] [Q&A] [answered]
For example, let's say I have a road segmentation and I will have the annotations for an image and the next image is very close to the first one so I would like just to take the previous image and adapt them a bit for the new image.

I know it works for the same image to re add annotations, but I am interested for different images. 

I get this error: 
 _`CVAT Error: Request failed with status code 500. Exception: Could not match item id: 'name_item' with any task frame\n`_

Thanks in advance

**Comment by bsekachev** [comment, marked as answer]
Hi, @vasigiurgi 

On UI you can use "propagate" feature to copy annotation from current frame to any number of frames ahead. 
![Screenshot from 2021-03-22 11-37-54](https://user-images.githubusercontent.com/40690378/111962231-19444780-8b03-11eb-9c16-1e6e59ce8b28.png)

It works for one selected object, so, if you need to copy a lot of them, no straight way to do that, only propagating each object separately.
